src/components/Router.js

```diff
@@ -42,7 +42,7 @@ class Router extends Component {
   render() {
     return (
       <div>
-        {this.props.page === 'login' ? "" : <Nav events={this.state.events} />}
+        {this.props.user ? <Nav events={this.state.events} /> : null }
```

Comment:
This is related to your other comment.
I changed the logic so instead of checking if the page is `login` (user is not logged in yet, previously the page was set to `home` once user was logged in), we check if the `user` object exists/is populated. If it is, display the navbar, else display nothing. Events are props passed to navbar